src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -82,9 +132,25 @@ export class TablePaginationControls extends React.Component<ITablePaginationCon
                     <Button key="prevPageBtn" disabled={!!this.props.previousPageDisabled} onClick={this.props.onPreviousPageClick}>
                         {this.props.previousButtonContent}
                     </Button>
-                    <Button key="textBetweenButtons" className={styles["auto-cursor"]} disabled={true}>
-                        {this.props.textBetweenButtons}
-                    </Button>
+                    <If condition={!this.textBoxActive}>
+                        <Then>
+                            <Button key="textBetweenButtons" className={styles["auto-cursor"]} disabled={!this.props.pageNumberEditable}
```

Comment:
but the disabled button says `1 of 61  patients`, so it doesn't make sense to make the whole text editable. when user clicks the button(not disabled anymore), it turns to a text input with only `1` in it instead of `1 of 61 patients`, and in the event of enter key press or onBlur, it navigates to the new index and text input turns back to a button. you will see what I mean when you use it.